### DISCUSSION TITLE
Start/Stop rustdesk.service instead of Enable/Disable

### DISCUSSION THREAD

**Discussion by Lutzx** [rustdesk/rustdesk#6480] [Feature Request]
It would be nice to have an option to start the service on the GUI instead of enabling it for future boots on Linux. And the same for closing the tray, there should be an option to only stop the current service and not disable it for future boots. I am using this with elderly family members that sometimes forget they can't close the system tray and then I can't connect remotely to help

**Comment by Selaron** [comment]
What is that service actually doing? I'm using rustdesk linux client to connect an remote control family members' computers in order to offer assistance.
There does not seem to be a use case for me so I disabled that service.
